app/code/Magento/WishlistGraphQl/Model/Resolver/UpdateProductsInWishlist.php

```diff
@@ -160,4 +186,107 @@ private function getWishlist(?int $wishlistId, ?int $customerId): Wishlist
 
         return $wishlist;
     }
+
+    /**
+     * Update wishlist Item and set data from request
+     *
+     * @param int $itemId
+     * @param DataObject $buyRequest
+     * @param wishlistFactory $wishlist
+     * @param null $params
```

Comment:
null is not a valid type, you will get a phpcs error. Also I dont see params being used. Please handle it  accordingly.